elasticlib/Elastic.java

```diff
@@ -55,6 +74,17 @@ public String getTitle() {
       return title;
     }
 
+    public int getDisplayTime() {
+      return displayTime;
+    }
+
+    /*
+     * Display time in miliseconds
+     */
+    public void setDisplayTime(int displayTime) {
+      this.displayTime = displayTime;
+    }
```

Comment:
These should probably specify milliseconds somewhere in the method/variable name to avoid confusion